doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_nordic_bundles.rst

```diff
@@ -0,0 +1,57 @@
+.. _ug_nrf54h20_suit_nordic_bundles:
```

Comment:
```suggestion
.. _ug_nrf54h20_suit_soc_binaries:
```

Reply:
Just trying to keep the approved product name (nRF54H20 SoC binaries) as consistent as possible across documents